plugin/src/main/java/org/wildfly/plugin/provision/ApplicationImageMojo.java

```diff
@@ -95,6 +97,9 @@ public class ApplicationImageMojo extends PackageServerMojo {
     @Parameter(alias = "image")
     private ApplicationImageInfo image;
 
+    @Parameter(property = WILDFLY_IMAGE_LABELS, alias = "labels", required = false)
+    private List<String> labels = Collections.emptyList();
```

Comment:
Should this be a `Map<String, String>` instead so it can be defined like:
```xml
<labels>
    <version>1.0</version>
    <description>&quot;Test description &quot;</description>
</labels>
```

Also the alias and required aren't really needed.